### ISSUE TITLE
Critical Dependencies Warnings

### ISSUE THREAD

**Issue by chobo2** [euforic/banking.js#47]
Hi

I am getting these areas

```
WARNING in ./~/formidable/lib/incoming_form.js
Critical dependencies:
1:43-50 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/formidable/lib/incoming_form.js 1:43-50

WARNING in ./~/formidable/lib/file.js
Critical dependencies:
1:43-50 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/formidable/lib/file.js 1:43-50

WARNING in ./~/formidable/lib/querystring_parser.js
Critical dependencies:
1:43-50 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/formidable/lib/querystring_parser.js 1:43-50

WARNING in ./~/formidable/lib/json_parser.js
Critical dependencies:
1:43-50 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/formidable/lib/json_parser.js 1:43-50
```

**Comment by webmonarch** [comment]
Are you building with webpack? I just came across a similar warning when using `banking.js` in my project.  I've fixed the warning in `ofx.js` with #49 but it looks like formidable specifically redefines `require` for some "clever" purpose that Webpack doesn't like.  See https://github.com/felixge/node-formidable/blob/master/lib/incoming_form.js#L1

I've avoided this warning in my use-case by importing `ofx.js` directly (I only need OFX parsing).
